Documentation/contributing.rst

```diff
@@ -55,6 +55,39 @@ requirements have been met:
 * You have signed off on your commits, see the section "Developer's
   Certificate of Origin" for more details.
 
+Release Process
+---------------
+
+Cilium schedules a major release every 3 months. Each major release is
+performed by incrementing the `Y` in the version format `X.Y.0`. The group of
+committers can decide to increment `X` instead to mark major milestones in
+which case `Y` is reset to 0.
+
+The following steps are performed to publish a release:
+
+1. The master branch is set to the version `X.Y.90` at all times. This ensures
```

Comment:
We already include the git SHA in the version information together with the go version used to build:
```
$ ./cilium/cilium version
Cilium 0.8.0 1185e7a Thu, 30 Mar 2017 02:18:25 +0200 go version go1.7.4 linux/amd64
```

The OVS project is using the `.90` version method and it seems to work fine for them.

The advantage of `x.y.90` over `x.y.z+` is that it is guaranteed that if someone is running `0.8.3` and wants to upgrade to the latest development snapshot `0.8.90`, that upgrade is painless and simple with packages built because the development version is always considered more recent because of the higher version number. Hence my question on how the `+` addition is interpreted by package management systems.